docs/src/main/tut/json.md

```diff
@@ -206,6 +216,13 @@ However, be cautious when using this. Having this implicit
 in scope does mean that we would always try to decode HTTP entities
 from JSON (even if it is XML or plain text, for instance).
 
+For more convenience there is import combining both encoding 
+and decoding derivation: 
+
+```tut:book
+org.http4s.circe.CirceEntityEncDec._
```

Comment:
This tut broke with an ambiguous implicit.  We need to sort this out.  Running `sbt doc/tutQuick` should test it locally.

Reply:
Woops, the exact thing I wrote: we can't use both `CirceEntityEncoder` and plain json. I [restricted the scope](https://github.com/http4s/http4s/pull/1917/commits/4d73de742785a58cb5753f427e07e115ad99ba3f#diff-dcf0079785763cffc8cb901dad9f0628R168), but I'm not sure if it's the right way. Maybe I should not use `tut:book` there at all?

Reply:
I think it's good to ensure that the import continues to compile, and you note the risk of ambiguity below.  I played with invisible braces around it, and it doesn't work.  I think the best we can do.